test/FastTests/Voron/Trees/TreeRenaming.cs

```diff
@@ -8,6 +8,7 @@
 using Xunit;
 using Voron.Global;
 using Xunit.Abstractions;
+using Elastic.Clients.Elasticsearch;
```

Comment:
Unintended?

Reply:
I hate resharper and it's automatic additions. Yes.